### PR TITLE
9211 Enable Opinion Search for External Users

### PR DESCRIPTION
**Pull request by jamesobrooks**
ustaxcourt/ef-cms#8143 
Enables Opinion search for external, logged-in users.
Adds integration tests ensuring that these users can perform the search properly.

### CODE REVIEW

File: web-client/src/views/UploadCourtIssuedDocument/UploadCourtIssuedDocument.jsx [hunk 1]
```diff
@@ -48,7 +48,7 @@ export const UploadCourtIssuedDocument = connect(
           <div className="grid-container padding-x-0">
             <div className="grid-row grid-gap">
               <div className="grid-col-12">
-                <h2 className="heading-1">Upload PDF</h2>
+                <h2 className="heading-1">Upload PDF!!!</h2>
```
Comment (mrinsin) [inline web-client/src/views/UploadCourtIssuedDocument/UploadCourtIssuedDocument.jsx]: Accident? :D

**Review by mrinsin — APPROVED** [review]
Loving these test cases!

**Review by TomElliottFlexion — APPROVED** [review]

File: web-client/src/views/UploadCourtIssuedDocument/UploadCourtIssuedDocument.jsx (hunk 1 above)
Comment (jamesobrooks) [reply web-client/src/views/UploadCourtIssuedDocument/UploadCourtIssuedDocument.jsx]: I wasn't sure why that was done. It looks like it came in with [this commit](https://github.com/flexion/ef-cms/pull/9288/commits/8b42137ec30d7d833509db871773024fcfc2fbd7) by @codyseibert.

Comment (codyseibert) [reply web-client/src/views/UploadCourtIssuedDocument/UploadCourtIssuedDocument.jsx]: @jamesobrooks 
accident, I'm removing it now

Comment (ujahio) [reply web-client/src/views/UploadCourtIssuedDocument/UploadCourtIssuedDocument.jsx]: It was for a test but never changed.

Comment (mrinsin) [reply web-client/src/views/UploadCourtIssuedDocument/UploadCourtIssuedDocument.jsx]: Maybe it's just leftover from when someone was trying to figure out if they were working on the right JSX file? I will defer to Cody!

Comment (mrinsin) [reply web-client/src/views/UploadCourtIssuedDocument/UploadCourtIssuedDocument.jsx]: NVM i see he removed it!

**Pull request merged**